index.html

```diff
@@ -14,9 +14,9 @@ <h1>todo</h1>
       <div id="todoList"></div>
     </div>
 
-    <script src="./components/button/buttton.js"></script>
-    <script src="./components/todo/todoInput.js"></script>
-    <script src="./components/todo/todoItem.js"></script>
-    <script src="./js/app.js"></script>
+    <script type="module" src="./components/button/button.js"></script>
+    <script type="module" src="./components/todo/todoInput.js"></script>
+    <script type="module" src="./components/todo/todoItem.js"></script>
+    <script type="module" src="./js/app.js"></script>
```

Comment:
lets import this component.

like `import './components....'` in the component where needed.